www/components/shelf/shelf.js

```diff
@@ -50,7 +50,8 @@ class Shelf extends LitElement {
   expandRoute(path) {
     // find list item containing current window.location.pathname
     let routeShelfListIndex = this.shelfList.findIndex(list => {
-      return list.item.link.indexOf(path) >= 0;
+      let expRoute = new RegExp(`^${path}$`);
```

Comment:
ah... I think this has been a solution i've been looking for, wherever we have to do this sort of pattern matching for current route like in [app-template.js ](https://github.com/ProjectEvergreen/greenwood/blob/master/packages/cli/src/templates/app-template.js#L26) and could help fix our [missing forward `/` issue](https://trello.com/c/AuzJwHkZ)?

Basically anywhere else we are doing something like this
```
const route = window.location.pathname;

const currentPage = response[1].data.graph.filter((page) => {
  return route === page.link;
})[0];
```